### DISCUSSION TITLE
More crypto support

### DISCUSSION THREAD

**Discussion by GomorrhaDev** [Paymenter/Paymenter#194] [Ideas]
Currently only very few cryptocurrencies are supported, which all have really high transaction fees!
Therefore, I suggest to add more cryptocurrencies, and therefore add a gateway such as https://nowpayments.io/, as this also has relatively low fees (0.5% if you do not want to activate additional features). Also, with nowpayments you have the possibility to exchange the cryptocurrencies automatically or to transfer them as fiat currency to a bank account.

**Comment by 09han09** [comment]
https://documenter.getpostman.com/view/7907941/2s93JusNJt#intro

^^
If someone wants to look further into it here are the docs already.

**Reply by cryptoloveearning** [reply]
I will thanks

**Comment by GH0st3rs** [comment]
Hi You can try to use my integration.
https://github.com/GH0st3rs/NOWpayments-paymenter

Fill free to submit Issue and PR
